app/src/lib/design-system/components/RQNetworkTable/GenericNetworkTable/components/FiltersToolbar/FiltersToolbar.tsx

```diff
@@ -12,9 +13,34 @@ interface Props {
 
 const FiltersToolbar: React.FC<Props> = ({ filters, setFilters }) => {
   const [searchValue, setSearchValue] = useState(filters?.search ?? "");
+  const [statusCodeFilters, setStatusCodeFilters] = useState(filters?.statusCode ?? []);
+  const [methodFilters, setMethodFilters] = useState(filters?.method ?? []);
+
   const onSearchFilterChange = (searchValue: string) => {
     setFilters({ ...filters, search: searchValue });
   };
+
+  const onGroupFilterChange = useCallback(
+    (filterValue: string, filterType: string) => {
+      const currentFilter = filterType === FilterKeys.STATUS_CODE ? statusCodeFilters : methodFilters;
+
+      if (currentFilter.indexOf(filterValue) !== -1) {
+        currentFilter.splice(currentFilter.indexOf(filterValue), 1);
+      } else {
+        currentFilter.push(filterValue);
+      }
```

Comment:
Instead of adding this logic here, we can directly send the final value `['GET', 'POST']` to this method from MethodFilter component and keep this logic there.